scripts/gulp-start-gae-devserver.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This originated from this repo https://github.com/maciejzasada/gulp-gae.
+ * We do not use it directly because supported actions are
+ * dev_appserver.py and appcfg.py which will require every developer to install
+ * google app_engine and export its path.
+ *
+ * @fileoverview module that start gae server in gulp.
+ *
+ * @author barnabasmakonda@gmail.com (Barnabas Makonda)
+ */
+
+'use strict';
+
+var gulpUtil = require('gulp-util');
+var path = require('path');
+var spawn = require('child_process').spawn;
+var through = require('through2');
+
+var File = gulpUtil.File;
+var PluginError = gulpUtil.PluginError;
+
+module.exports = function(action, args, params) {
+  action = action || defaultStartAction;
```

Comment:
FYI I tried setting this up and got a "defaultStartAction is not defined" error.